options.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<head>
+<meta charset="utf-8">
+</head>
+<body>
+<label>
+  <input type="checkbox" id="prefPrependSnowflake">
+  <!-- options.js will add a label with i18n ID toggleSnowflakePref. -->
```

Comment:
Can we put the text in a `<span>` that defaults to hardcoded English and gets changed to the localized version via script? Replacing `textContent` via the script might save us a layout reflow (and give a nicer experience to the default locale)

Reply:
Done. Initially I did not do so to not duplicate the string, but since you've requested it, I have amended the commit. This is the difference:

```diff
diff --git a/options.html b/options.html
index 9eea240..a9c2d09 100644
--- a/options.html
+++ b/options.html
@@ -5,7 +5,8 @@
 <body>
 <label>
   <input type="checkbox" id="prefPrependSnowflake">
-  <!-- options.js will add a label with i18n ID toggleSnowflakePref. -->
+  <!-- options.js will replace the label with i18n ID toggleSnowflakePref. -->
+  <span id="i18n-toggleSnowflakePref">Prepend snowflake to discarded tabs (requires extra permission).</span>
 </label>
 <script src="options.js"></script>
 </body>
diff --git a/options.js b/options.js
index 173c3f0..57cdc40 100644
--- a/options.js
+++ b/options.js
@@ -18,8 +18,8 @@ prefPrependSnowflake.onchange = async () => {
   });
 };
 
-prefPrependSnowflake.insertAdjacentText("afterEnd",
-  browser.i18n.getMessage("toggleSnowflakePref"));
+document.getElementById("i18n-toggleSnowflakePref").textContent =
+  browser.i18n.getMessage("toggleSnowflakePref");
 
 browser.storage.onChanged.addListener((changes) => {
   if (changes.prependSnowflake) {
```